static/files/postinst.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# add sensu group
+set +e
+grep "^sensu:" /etc/group > /dev/null
+RC=$?
+set -e
+if [ "${RC}" != 0 ]; then
+    groupadd sensu
+fi
+
+# add sensu group
+set +e
+grep "^sensu:" /etc/passwd > /dev/null
```

Comment:
We've previously been called out for grepping /etc/passwd instead of using `getent`. Is that applicable here as well?